test/ui-pytest/gui/screens/settings.py

```diff
@@ -75,3 +76,37 @@ def get_community_info(self, name: str) -> UserCommunityInfo:
     def open_community_overview_settings(self, name: str):
         driver.mouseClick(self._get_community_item(name))
         return CommunitySettingsScreen().wait_until_appears()
+
+
+class KeycardSettingsView(QObject):
+
+    def __init__(self):
+        super(KeycardSettingsView, self).__init__('mainWindow_KeycardView')
+        self._scroll = Scroll('settingsContentBaseScrollView_Flickable')
+        self._setup_keycard_with_existing_account_button = Button(
+            'settingsContentBaseScrollView_setupFromExistingKeycardAccount_StatusListItem')
+        self._create_new_keycard_account_button = Button(
+            'settingsContentBaseScrollView_createNewKeycardAccount_StatusListItem')
+        self._import_restore_via_seed_phrase_button = Button(
+            'settingsContentBaseScrollView_importRestoreKeycard_StatusListItem')
+        self._import_from_keycard_button = Button('settingsContentBaseScrollView_importFromKeycard_StatusListItem')
+        self._check_whats_on_keycard_button = Button(
+            'settingsContentBaseScrollView_checkWhatsNewKeycard_StatusListItem')
+        self._factory_reset_keycard_button = Button('settingsContentBaseScrollView_factoryResetKeycard_StatusListItem')
+
```

Comment:
Please make symbolic names shorter and maybe names of variables.
It is good to fir one class attribute in one line.

Reply:
Done